packages/terra-clinical-result/README.md

```diff
@@ -0,0 +1,33 @@
+# Terra Clinical Result
+
+
+[![NPM version](https://badgen.net/npm/v/terra-clinical-result)](https://www.npmjs.org/package/terra-clinical-result)
+[![Build Status](https://badgen.net/travis/cerner/terra-clinical)](https://travis-ci.com/cerner/terra-clinical)
+
+The terra-clinical-onset-picker component provides users a way to enter or select an approximate date for onset scenarios.
```

Comment:
Is the the description for clinical-result? Sounds like it might have been missed and copied from onset picker on accident

Reply:
Oops updated here https://github.com/cerner/terra-clinical/pull/628/commits/913f6ab0620a3d1d8e10b27a48884696f2881ef0.